main/gdm/template.py

```diff
@@ -55,6 +55,7 @@
     "gnome-settings-daemon",
     "gnome-shell",
     "gsettings-desktop-schemas",
+    "iio-sensor-proxy-meta",
```

Comment:
does gdm use it? i couldn't find anything for it

Reply:
(in contrast to g-c-c which looks stuff up in net.hadess.SensorProxy, etc)

Reply:
ambient light sensor certainly reacts for me there and adjusts the brightness, that was kinda my basis for adding it there since I still can't test auto-rotation until my accelerometer sensor on the tablet is functional on linux

Reply:
sure, but it might not do it directly? there are zero references to 'sensor' in gdm

Reply:
from what i can tell it's g-s-d that does things like that and probably automatically changes the brightness in settings

Reply:
sure I'll move it to be a dep of `gnome-settings-daemon` instead